### DISCUSSION TITLE
Integration With Redux

### DISCUSSION THREAD

**Discussion by trinadhkoya** [vercel/next.js#19440] [Ideas]
As i can see, it has already taken care of React. How about adding Redux as well to make it easy for fellow developers

**Comment by kachkaev** [comment]
Next.js repo includes a bunch of official examples:
https://github.com/vercel/next.js/tree/canary/examples

Several of them contain redux (search the list by name). The most pure example with Redux is:
https://github.com/vercel/next.js/tree/canary/examples/with-redux
